File: wrapUpTasks/javaScript/countLeapYearsBetween1900To2025.js

let startingYear = 1900;
let endingYear = 2025;
let count = 0;

for(let year = startingYear; year <= endingYear; year++){
    
    if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)){
        
        count++;
    }
}
console.log(count);

File: wrapUpTasks/java/CountLeapYearsBetween1900To2025.java
public class CountLeapYearsBetween1900To2025{
    
    public static void main(String[] args){
        
        int startingYear = 1900;
        int endingYear = 2025;
        int count = 0;
        
        for(int year = startingYear; year <= endingYear; year++){
            
            if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)){

                count++;
            }
        }
        System.out.println(count);
        
    }
}
